src/main/scala/gitbucket/core/controller/api/ApiIssueCommentControllerBase.scala

```diff
@@ -65,9 +65,42 @@ trait ApiIssueCommentControllerBase extends ControllerBase {
   })
 
   /*
-   * v. Edit a comment
-   * https://developer.github.com/v3/issues/comments/#edit-a-comment
+   * v. Update an issue comment
+   * https://developer.github.com/v3/issues/comments/#update-an-issue-comment
    */
+  patch("/api/v3/repos/:owner/:repository/issues/comments/:id")(readableUsersOnly { repository =>
+    val commentId = params("id")
+    getComment(repository.owner, repository.name, commentId) match {
+      case Some(issueComment) =>
+        getIssue(repository.owner, repository.name, issueComment.issueId.toString) match {
+          case Some(issue) =>
+            if (isEditable(issue.userName, issue.repositoryName, issue.openedUserName)) {
```

Comment:
We need to check whether a comment is editable, not an issue.

Reply:
@takezoe  Thx for your review! I modified PR at https://github.com/gitbucket/gitbucket/pull/2538/commits/38bd5d912ce8f793abbbb1ecbef85438400eb7a3.